tests/contrib/flask/flask_tests.py

```diff
@@ -144,11 +145,23 @@ def test_instrumentation_404(flask_apm_client):
 
     assert len(transactions) == 1
     traces = transactions[0]['traces']
-    print(transactions[0])
-    assert transactions[0]['result'] == '404'
+    assert transactions[0]['result'] == 'HTTP 4xx'
+    assert transactions[0]['context']['response']['status_code'] == 404
     assert len(traces) == 0, [t["signature"] for t in traces]
 
 
+def test_non_standard_http_status(flask_apm_client):
+    with mock.patch("elasticapm.traces.TransactionsStore.should_collect") as should_collect:
+        should_collect.return_value = False
+        resp = flask_apm_client.app.test_client().get('/non-standard-status/')
+    assert resp.status == "0 fail", resp.response
+    assert resp.status_code == 0, resp.response
+
+    transactions = flask_apm_client.client.instrumentation_store.get_all()
+    assert transactions[0]['result'] == '0 fail'  # "0" is prepended by Werkzeug BaseResponse
```

Comment:
Does this mean that every `transaction.result` not coming from a `flask` or `django` app has a `0 ` prefix no matter what? Could you get rid of this somehow?

Reply:
Not exactly. It means that if a flask user returns a non-integer HTTP status code (which is all kinds of wrong in the first place), flask will prepend this `0` to it. Not sure why, but I'm not hugely concerned about it, since it only happens when the user code doing something completely wrong

Reply:
oh yes, it is a flask test.. then I am with you to not worry about this too much.